### ISSUE TITLE
PEP 541 Request: django-rest-admin

### ISSUE THREAD

**Issue by ZeyadMoustafaKamal** [pypi/support#4709]
### Project to be claimed

`django-rest-admin`: https://pypi.org/project/django-rest-admin

### Your PyPI username

`zeyadmoustafa`: https://pypi.org/user/zeyadmoustafa

### Reasons for the request

well the current project maintenance haven't update the package since a year and I think that this is a good reason to take the project name. but the other thing is that the package name is `django-rest-admin` and it tried to create a project called `djangorestadmin` but I was told that the name is similer to an existing one.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

the current is https://github.com/Jiangshan00001/django_rest_admin

and the one I will use is https://github.com/ZeyadMoustafaKamal/django-rest-admin but please note that I currently maintain it locally to be ready to provide the basic functionality and that I won't publish it on pypi until I finish this basic functionality.

### Contact and additional research

the project maintainer haven't opened github since a year and I didn't get any kind of email or something like that in their github profile or in their website.

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @ZeyadMoustafaKamal,

We are working to contact the owner to determine if they're reachable and have sent them a courtesy message.

Final determination regarding project ownership will be made by the PyPI Administrators when the research process is complete.

**Comment by Thespi-Brain** [comment]
Hi @ZeyadMoustafaKamal,

Before we can transfer the project over to you, we took a look at your replacement repo. Your project seems like it's a work in progress and in order for us to transfer it to you, we do need proof that you are fully developing something complete. Is there another repo where you are developing your project that we can take a look at? thanks.

**Comment by ZeyadMoustafaKamal** [comment]
@Thespi-Brain yes I have actually stopped working in that project right now because of my limited time. But I do plan to continue working in it in the future. So can we just leave the issue opened until I finally complete it ?

**Comment by Thespi-Brain** [comment]
Hello @ZeyadMoustafaKamal,

You can create a new issue once you decide you want to work on the project. For now, I am closing this issue.
